maistra/ci/common.sh

```diff
@@ -48,3 +56,10 @@ function bazel_test() {
 
 # Fix path to the vendor deps
 sed -i "s|=/work/|=$(pwd)/|" maistra/bazelrc-vendor
+
+# Create symbolic links to existent executables in the builder
+mkdir -p "$WORKDIR/maistra/local/bin"
+ln -f -s /bin/node $WORKDIR/maistra/local/bin/node
```

Comment:
I recommend installing the `shellcheck` plugin for vscode. It would catch things like this. You need to put quotes around shell variables.